py/selenium/webdriver/common/selenium_manager.py

```diff
@@ -0,0 +1,43 @@
+# Licensed to the Software Freedom Conservancy (SFC) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The SFC licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import logging
+import os
+import subprocess
+
+logger = logging.getLogger(__name__)
+
+
+class SeleniumManager:
+    """
+    Wrapper for getting information from the Selenium Manager binaries.
+    """
+
+    @classmethod
+    def driver_location(cls, browser: str) -> str:
```

Comment:
is a simple function suitable here rather than the class overhead? `@classmethod` seems abusive as it cares not about the `cls` itself / acting like some sort of factory, it's also only used once so making it just a function of the class normally and setting it as a `@property` would be sufficient too - if you think we might build on this in future and require internal state etc, otherwise a simple function is this module is perfect.

Reply:
I use class methods any time I have a function that doesn't require state. The instance method in the Service class has the state and is calling out for a static functional response. I'm expecting we'll also be implementing a `browser_location()` method as well at some point. If we implement a method to download the latest Selenium Manager for the given OS, though, that will probably set a state. So I'm interested in seeing what @bonigarcia thinks is a good way to do this in Java.

Reply:
we should make this a `@staticmethod` then rather than `@classmethod`

Reply:
Ah, yes that's what I want to do for Python. Thanks.